com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/MotionBlur.compute

```diff
@@ -127,7 +128,7 @@ void MirrorWeights(float depth1, float depth2, float motionVec1, float motionVec
 // Sample processing 
 // ---------------------------------------------
 
-float3 ProcessSampleFastPath(uint sampleNumber, float2 dir, float invSampleCount, float2 centralUV, float randomVal, float dirSign)
+float3 ProcessSampleFastPathF3(uint sampleNumber, float2 dir, float invSampleCount, float2 centralUV, float randomVal, float dirSign)
```

Comment:
you can do is more simply

#pragma kernel MotionBlurF3CS 
#pragma kernel MotionBlurF4CS ENABLE_ALPHA

#if ENABLE_ALPHA
#define ProcessSampleFastPath ProcessSampleFastPathF4
#else
#define ProcessSampleFastPath ProcessSampleFastPathF3
#endif

Reply:
This avoid to duplicate all this code

Reply:
You are right.
How do you think about this fix?

As far as I've tested it is working.

```
#include "Packages/com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/MotionBlurCommon.hlsl"

#pragma kernel MotionBlurF3CS
#pragma kernel MotionBlurF4CS ENABLE_ALPHA

#if ENABLE_ALPHA
#define CTYPE float4
#define CTYPE_SWIZZLE xyzw
#else
#define CTYPE float3
#define CTYPE_SWIZZLE xyz
#endif
//#pragma enable_d3d11_debug_symbols

RW_TEXTURE2D_X(CTYPE, _OutputTexture);


void KernelMain(uint3 dispatchThreadId ) // 
{
    UNITY_XR_ASSIGN_VIEW_INDEX(dispatchThreadId.z);
    CTYPE debugColor = 1.0f;
    // Get central sample early.
    CTYPE centralColor = _InputTexture[COORD_TEXTURE2D_X(dispatchThreadId.xy)].CTYPE_SWIZZLE;
     .
     .
     .
     .
     .
}
// ---------------------------------------------
// Kernel
// ---------------------------------------------
[numthreads(16, 16, 1)]
void MotionBlurF4CS(uint3 dispatchThreadId : SV_DispatchThreadID)
{
    KernelMain(dispatchThreadId);
}
[numthreads(16, 16, 1)]
void MotionBlurF3CS(uint3 dispatchThreadId : SV_DispatchThreadID)
{
    KernelMain(dispatchThreadId);
}

```

Reply:
I committed this and started yamato test.